samza-core/src/main/java/org/apache/samza/clustermanager/ContainerProcessManager.java

```diff
@@ -236,6 +237,12 @@ public void start() {
     Map<String, String> processorToHostMapping = state.jobModelManager.jobModel().getAllContainerLocality();
     containerAllocator.requestResources(processorToHostMapping);
 
+    // Initialize the per processor failure count to be 0
+    processorToHostMapping.keySet().forEach(processorId -> {
```

Comment:
See comment below on how/why this information isnt really in "state"

Reply:
replied there